### ISSUE TITLE
`fab plugin add`

### ISSUE THREAD

**Issue by geelen** [fab-spec/fab#199]
Need a way to quickly either drop in a plugin from NPM or create a new local file in typescript with the right import and a quick and dirty template.

**Comment by evanderkoogh** [comment]
Yeah, the use-case I had in mind was to very quickly create a scaffold for a server-side file. So that we can really hit home how easy it is to go from a static site to a site with some server-side code sprinkled in.
